src/v2/Apps/ViewingRoom/Components/ViewingRoomsFeaturedRail.tsx

```diff
@@ -16,19 +27,105 @@ export const ViewingRoomsFeaturedRail: React.FC<ViewingRoomsFeaturedRailProps> =
     })
     .filter(Boolean)
 
-  return featuredViewingRoomsForRail.length > 0 && (
-    <Box>
-      <Sans size="5">Featured</Sans>
+  const computeScrollPercent = selectedIndex =>
+    ((selectedIndex + 1) / featuredViewingRoomsForRail.length) * 100
+  const [scrollPercent, setScrollPercent] = useState(computeScrollPercent(0))
+  const update = flowRight(setScrollPercent, computeScrollPercent)
+  const showProgressBar = featuredViewingRoomsForRail.length > 1
+
+  const CarouselHeight = 380
+
+  return (
+    featuredViewingRoomsForRail.length > 0 && (
       <Box>
-        {featuredViewingRoomsForRail.map(vr => {
-          return (
-            <Sans size="3t" key={vr.slug}>
-              {vr.title}
-            </Sans>
-          )
-        })}
+        <Sans size="5">Featured</Sans>
+        <Box width="100%">
+          <Flex
+            height={CarouselHeight}
+            // maxWidth={breakpoints.lg}
+            m="auto"
+            my={2}
+            position="relative"
+            justifyContent="left"
+          >
+            <Carousel
+              options={{
+                cellAlign: "center",
+                draggable: showProgressBar,
+                freeScroll: true,
+                groupCells: 1,
+                pageDots: false,
+              }}
+              data={featuredViewingRoomsForRail}
+              height={CarouselHeight}
+              onDragEnd={({ flickity }) => update(flickity.selectedIndex)}
+              render={(
+                {
+                  heroImageURL,
+                  title,
+                  partner,
+                  status,
+                  distanceToOpen,
+                  distanceToClose,
+                },
+                slideIndex
+              ) => {
+                const tag = getTagProps(status, distanceToOpen, distanceToClose)
+                return (
+                  <Flex flexDirection="row">
+                    {slideIndex !== 0 && <Spacer ml="15px" />}
```

Comment:
Maybe no need for spacer after the last element either?

Reply:
Hmm wouldn't this only ever put a spacer in front of the card? so there's no spacer after the last element?